code/datums/status_effects/debuffs.dm

```diff
@@ -659,6 +659,12 @@
 			dreamer.adjustBruteLoss(-1, FALSE)
 			dreamer.adjustFireLoss(-1, FALSE)
 			dreamer.adjustToxLoss(-1)
+			dreamer.adjustOxyLoss(-1)
+			dreamer.adjustCloneLoss(-0.5)
+			if(dreamer.HasDisease(/datum/disease/critical/heart_failure))
+				if(prob(25))
```

Comment:
instead of adding another if statement and indentation, I would just tack this onto the previous if statement `&&`

Reply:
this mean coffin will heal you only when you are in crit, and not allowing you to heal manually. I dont think this is good idea. Or i simply dont understand you

Reply:
I just mean condense the if statements:
`			if(dreamer.HasDisease(/datum/disease/critical/heart_failure) && prob(25))`

Reply:
oh, yes